falcon/asgi/response.py

```diff
@@ -14,18 +14,18 @@
 
 """ASGI Response class."""
 
-from asyncio.coroutines import CoroWrapper  # type: ignore
-from inspect import iscoroutine, iscoroutinefunction
+from asyncio.coroutines import CoroWrapper
+from inspect import iscoroutine
+from inspect import iscoroutinefunction
 
+from falcon import response
 from falcon.constants import _UNSET
-import falcon.media
```

Comment:
did this had a particular meaning? it's not used

Reply:
Probably a leftover from different phases of refactoring